scio-test/src/main/scala/com/spotify/scio/testing/ContextProvider.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2016 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.testing
+
+import com.spotify.scio.ScioContext
+
+private[testing] trait ContextProvider {
+  def context: ScioContext
+}
+
+private[testing] trait UnitTestContextProvider extends ContextProvider {
+  override def context: ScioContext = ScioContext.forTest()
+}
+
+private[testing] trait IntegrationTestContextProvider extends ContextProvider {
+  override def context: ScioContext = ScioContext()
```

Comment:
This is a bit misleading since it's still a local runner and doesn't actually run on any service.